darts/models/forecasting/torch_forecasting_model.py

```diff
@@ -81,6 +81,10 @@
 
 logger = get_logger(__name__)
 
+# Check whether we are running pytorch-lightning >= 1.7.0 or not:
+tokens = pl.__version__.split(".")
+pl_170_or_above = int(tokens[0]) >= 1 and int(tokens[1]) >= 7
```

Comment:
Where is this used?

Reply:
it was used before I removed the torch_device_str. Its now removed